### PR TITLE
created Circle CI image checker

### PR DESCRIPTION
**Pull request by AndrewJMD**
If there is a link in an image it replaces it fails the check, all previous image links were fixed

### CODE REVIEW

File: build/checks/images.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# coding=utf8
+# the above tag defines encoding for this document and is for Python 2.x compatibility
+
+import sys
+import os
+import re
+import requests
```
Comment (gemohandy) [inline build/checks/images.py]: Apparently, this isn't a module. Are you sure it should be imported? It's causing issues.

File: build/checks/images.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+                                                else:
+                                                        print('Web site does not exist') 
+                                        except Exception as e:
+                                                print(e)             
+
+check = ExternalImages()
+check.ret = 0
+check.run()
+
+sys.exit(check.ret)
```
Comment (synixebrett) [inline build/checks/images.py]: Ending new line

File: www/artists/jmitchell2/discography.html [hunk 3]
```diff
@@ -8,13 +8,13 @@
     <body>
         <div class='content'>
             <h1>
-                Discography ♥
```
Comment (synixebrett) [inline www/artists/jmitchell2/discography.html]: This should have been replaced with a heart, not removed

File: www/artists/wtunison/collabor/ph000.htm [hunk 4]
```diff
@@ -177,7 +177,7 @@ <h1>
                 </td>
                 <td colspan="2">
                     <small>Visitors Since Jan 20/97:</small> <span class="c2"><span class="c1"><!-- change the address for the counter -->
-                     <img src="http://www2.regina.ism.ca/cgi-bin/xcount.cmd/artists/wtunison/?/artists/wtunison/collabor/ph000.htm/+1+5" align="bottom" alt="no alt text"></span></span>
+                     <img src="+1+5" align="bottom" alt="no alt text"></span></span>
```
Comment (synixebrett) [inline www/artists/wtunison/collabor/ph000.htm]: Hit counter

File: www/artists/wtunison/collabor/pv000.htm [hunk 5]
```diff
@@ -117,7 +117,7 @@ <h1>
                 </td>
                 <td colspan="2">
                     <small>Visitors Since Jan 20/97:</small> <span class="c2"><span class="c1"><!-- change the address for the counter -->
-                     <img src="http://www2.regina.ism.ca/cgi-bin/xcount.cmd/artists/wtunison/?/artists/wtunison/collabor/pv000.htm/+1+5" align="bottom" alt="no alt text"></span></span>
+                     <img src="+1+5" align="bottom" alt="no alt text"></span></span>
```
Comment (synixebrett) [inline www/artists/wtunison/collabor/pv000.htm]: Hit counter

File: www/artists/wtunison/collabor/pv002.htm [hunk 6]
```diff
@@ -163,7 +163,7 @@ <h1>
                 </td>
                 <td colspan="2">
                     <small>Visitors Since Jan 20/96:</small> <span class="c1"><span class="c1"><!-- change the address for the counter -->
-                     <img src="http://www2.regina.ism.ca/cgi-bin/xcount.cmd/artists/wtunison/?/artists/wtunison/collabor/ph000.htm/+1+5" align="bottom" alt="no alt text"></span></span>ison/collabor/ph000.htm/+1+5" ALIGN="bottom"&gt;
+                     <img src="+1+5" align="bottom" alt="no alt text"></span></span>ison/collabor/ph000.htm/+1+5" ALIGN="bottom"&gt;
```
Comment (synixebrett) [inline www/artists/wtunison/collabor/pv002.htm]: Hit counter

File: www/artists/wtunison/collabor/rules.htm [hunk 7]
```diff
@@ -131,7 +131,7 @@ <h1>
                 </td>
                 <td colspan="2">
                     <small>Visitors Since Jan 23/97:</small> <span class="c1"><span class="c1"><!-- change the address for the counter -->
-                     <img src="http://www2.regina.ism.ca/cgi-bin/xcount.cmd/artists/wtunison/?/artists/wtunison/collabor/rules.htm/+1+5" align="bottom" alt="no alt text"></span></span>
+                     <img src="+1+5" align="bottom" alt="no alt text"></span></span>
```
Comment (synixebrett) [inline www/artists/wtunison/collabor/rules.htm]: Hit counter

File: www/artists/wtunison/collabor/sc000.htm [hunk 8]
```diff
@@ -103,7 +103,7 @@ <h1>
                 </td>
                 <td colspan="2">
                     <small>Visitors Since Jan 20/97:</small> <span class="c2"><span class="c1"><!-- change the address for the counter -->
-                     <img src="http://www2.regina.ism.ca/cgi-bin/xcount.cmd/artists/wtunison/?/artists/wtunison/collabor/sc000.htm/+1+5" align="bottom" alt="no alt text"></span></span>
+                     <img src="+1+5" align="bottom" alt="no alt text"></span></span>
```
Comment (synixebrett) [inline www/artists/wtunison/collabor/sc000.htm]: Hit counter

File: www/artists/wtunison/collabor/sc002.htm [hunk 9]
```diff
@@ -113,7 +113,7 @@ <h1>
                 </td>
                 <td colspan="2">
                     <small>Visitors Since Jan 20/97:</small> <span class="c1"><span class="c1"><!-- change the address for the counter -->
-                     <img src="http://www2.regina.ism.ca/cgi-bin/xcount.cmd/artists/wtunison/?/artists/wtunison/collabor/sc002.htm/+1+5" align="bottom" alt="no alt text"></span></span>
+                     <img src="+1+5" align="bottom" alt="no alt text"></span></span>
```
Comment (synixebrett) [inline www/artists/wtunison/collabor/sc002.htm]: Hit counter

File: www/artists/wtunison/collabor/sd001.htm [hunk 10]
```diff
@@ -145,7 +145,7 @@ <h1>
                 </td>
                 <td colspan="2">
                     <small>Visitors Since Jan 20/97:</small> <span class="c2"><span class="c1"><!-- change the address for the counter -->
-                     <img src="http://www2.regina.ism.ca/cgi-bin/xcount.cmd/artists/wtunison/?/artists/wtunison/collabor/sd001.htm/+1+5" align="bottom" alt="no alt text"></span></span>
+                     <img src="+1+5" align="bottom" alt="no alt text"></span></span>
```
Comment (synixebrett) [inline www/artists/wtunison/collabor/sd001.htm]: Hit counter

File: www/artists/wtunison/collabor/tc001.htm [hunk 11]
```diff
@@ -92,7 +92,7 @@ <h1>
                 </td>
                 <td colspan="2">
                     <small>Visitors Since Jan 20/97:</small> <span class="c2"><span class="c1"><!-- change the address for the counter -->
-                     <img src="http://www2.regina.ism.ca/cgi-bin/xcount.cmd/artists/wtunison/?/artists/wtunison/collabor/tc001.htm/+1+5" align="bottom" alt="no alt text"></span></span>
+                     <img src="+1+5" align="bottom" alt="no alt text"></span></span>
```
Comment (synixebrett) [inline www/artists/wtunison/collabor/tc001.htm]: Hit counter

File: www/artists/wtunison/collabor/tc002.htm [hunk 12]
```diff
@@ -123,7 +123,7 @@ <h1>
                 </td>
                 <td colspan="2">
                     <small>Visitors Since Jan 20/97:</small> <span class="c2"><b>&gt;Visitors Since Jan 20/97:</b> <span class="c1"><span class="c1"><!-- change the address for the counter -->
-                     <img src="http://www2.regina.ism.ca/cgi-bin/xcount.cmd/artists/wtunison/?/artists/wtunison/collabor/tc002.htm/+1+5" align="bottom" alt="no alt text"></span></span></span>
+                     <img src="+1+5" align="bottom" alt="no alt text"></span></span></span>
```
Comment (synixebrett) [inline www/artists/wtunison/collabor/tc002.htm]: Hit counter

File: www/artists/wtunison/collabor/tc003.htm [hunk 13]
```diff
@@ -121,7 +121,7 @@ <h1>
                 </td>
                 <td colspan="2">
                     <small>Visitors Since Jan 20/96:</small> <span class="c1"><span class="c1"><!-- change the address for the counter -->
-                     <img src="http://www2.regina.ism.ca/cgi-bin/xcount.cmd/artists/wtunison/?/artists/wtunison/collabor/tc003.htm/+1+5" align="bottom" alt="no alt text"></span></span>
+                     <img src="+1+5" align="bottom" alt="no alt text"></span></span>
```
Comment (synixebrett) [inline www/artists/wtunison/collabor/tc003.htm]: Hit counter

File: www/artists/wtunison/collabor/ts000.htm [hunk 14]
```diff
@@ -106,7 +106,7 @@ <h1>
                 </td>
                 <td colspan="2">
                     <small>Visitors Since Jan 20/97:</small> <span class="c1"><span class="c1"><!-- change the address for the counter -->
-                     <img src="http://www2.regina.ism.ca/cgi-bin/xcount.cmd/artists/wtunison/?/artists/wtunison/collabor/ts000.htm/+1+5" align="bottom" alt="no alt text"></span></span>
+                     <img src="+1+5" align="bottom" alt="no alt text"></span></span>
```
Comment (synixebrett) [inline www/artists/wtunison/collabor/ts000.htm]: Hit counter

File: www/artists/wtunison/diary/intro.htm [hunk 15]
```diff
@@ -37,7 +37,7 @@ <h1>
             <tr>
                 <td>
                     <small>Visitors Since Jan 22/98:</small> <span class="c1"><span class="c1">
-                     <img src="http://www2.regina.ism.ca/cgi-bin/xcount.cmd/artists/wtunison/?/artists/wtunison/diary/intro.htm/+1+5" align="bottom" alt="no alt text"></span></span>
+                     <img src="+1+5" align="bottom" alt="no alt text"></span></span>
```
Comment (synixebrett) [inline www/artists/wtunison/diary/intro.htm]: Hit counter

**Pull request merged**